### DISCUSSION TITLE
モデル制限について

### DISCUSSION THREAD

**Discussion by RearAlice** [pixiv/vroid-sdk-developers#71] [Q&A] [answered]
いつも御世話になっております。

現在開発中のもので疑問点が出ましたので御相談したくDiscussion を立てさせていただきました。

変な化け物みたいなアバターが来たら場合によりゲームがバクる可能性があり、
(例としてこの変なアバターみたいなの→https://hub.vroid.com/characters/1929696511843428036/models/6805476078339054287)

利用規約以外でその変なモデルを弾くシステムってカスタマイズで構築したり規制したりする事はですか？

**Comment by nyamadan** [comment, marked as answer]
お世話になっております。

SDK自体にはモデルの利用規約以外で利用を制限する機能は存在しません。
一方で、[ポリゴン数](https://developer.vroid.com/sdk/docs/0.1.3/Pixiv.VroidSdk.Api.DataModel.CharacterModelProperty.html#Pixiv_VroidSdk_Api_DataModel_CharacterModelProperty_triangle_count)や[モデルサイズ](https://developer.vroid.com/sdk/docs/0.1.3/Pixiv.VroidSdk.Api.DataModel.CharacterModelVersion.html#Pixiv_VroidSdk_Api_DataModel_CharacterModelVersion_original_file_size)などを取得できるようになっておりますので、アプリケーションの仕様により必要であれば独自に制限を実装していただくことは可能です。
